src/styles/ChartSection.css

```diff
@@ -21,10 +22,55 @@
   width: 80%;
 }
 
-.chart-right {
```

Comment:
why was this removed?

Reply:
style cause problem with mobile styles so removed and refactored. Used grip gap instead of padding instead.

Reply:
approved changes